CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Changelog
 
+## [v0.5.0-dev.3](https://github.com/iotaledger/identity.rs/tree/v0.5.0-dev.3) (2022-01-24)
+
+[Full Changelog](https://github.com/iotaledger/identity.rs/compare/v0.5.0-dev.2...v0.5.0-dev.3)
+ 
+This release introduces a breaking change for diff updates created by versions `v0.5.0-dev.1` and `v0.5.0-dev.2` (previous diff updates from `<=v0.4.0` are already incompatible due to breaking changes to the document and message structure in `v0.5.0-dev.1`). To migrate, please publish an integration update containing all diff changes to prevent unexpected changes to resolved DID Documents.
+
+Changed
+
+- Remove `DocumentVerifier` [\#606](https://github.com/iotaledger/identity.rs/pull/606)
+- Fix dependent diff updates being rejected [\#605](https://github.com/iotaledger/identity.rs/pull/605)
+- Remove JSON string escaping in diff messages [\#598](https://github.com/iotaledger/identity.rs/pull/598)
+
+Added
+
+- Improve `OneOrMany` [\#602](https://github.com/iotaledger/identity.rs/pull/602)
+
+Patch
+
+- Fix diff properties \(de\)serialization [\#611](https://github.com/iotaledger/identity.rs/pull/611)
+
+**Removed:**
+
+- Change `Account::state` visibility to `pub(crate)` [\#604](https://github.com/iotaledger/identity.rs/pull/604)
```

Comment:
Do we want to use the `Removed` section or should it be combined with `Changed`?

If we do want it then the heading format should be changed to match.

Reply:
I think removed is valuable to have, which direction should we go with the formatting? I am leaning toward all with markdown `**`.

Reply:
Oh yeah, and with `:`?